dbms/src/Parsers/IdentifierQuotingStyle.h

```diff
@@ -8,9 +8,10 @@ namespace DB
 /// NOTE There could be differences in escaping rules inside quotes. Escaping rules may not match that required by specific external DBMS.
 enum class IdentifierQuotingStyle
 {
-    None,           /// Write as-is, without quotes.
-    Backticks,      /// `mysql` style
-    DoubleQuotes    /// "postgres" style
+    None,            /// Write as-is, without quotes.
+    Backticks,       /// `clickhouse` style
```

Comment:
Maybe we need to support ``` `quoted``identifiers` ``` in MySQL style in ClickHouse parser?

Reply:
Sure, but now ClickHouse uses `backslash` to escape `backquote`, and any changes could cause version-compatibility problems. Are we sure we want todo this?

Reply:
I forgot that we already do it:

```
:) SELECT 1 `a``b`

SELECT 1 AS `a\`b`                                                                                                                                                                                
                                                                                                                                                                                                  
┌─a`b─┐
│   1 │                                                                                                                                                                                           
└─────┘
```